### ISSUE TITLE
[Feature] Enable vela show to support parameter type "envMappings: [string]: [string]: string"

### ISSUE THREAD

**Issue by zzxwill** [oam-dev/kubevela#1520]
**Is your feature request related to a problem? Please describe.**

`vela show` can generate parameter table for complicated type, like `envMappings: [string]: [string]: string`. A complete TraitDefinition is as below.

```
apiVersion: core.oam.dev/v1beta1
kind: TraitDefinition
metadata:
  annotations:
    definition.oam.dev/description: "binding cloud resource secrets to pod env"
  name: service-binding
spec:
  appliesToWorkloads:
    - webservice
    - worker
  schematic:
    cue:
      template: |
        patch: {
        	spec: template: spec: {
        		// +patchKey=name
        		containers: [{
        			name: context.name
        			// +patchKey=name
        			env: [
        				for envName, v in parameter.envMappings {
        					name: envName
        					valueFrom: {
        						secretKeyRef: {
        							name: v.secret
        							if v["key"] != _|_ {
        								key: v.key
        							}
        							if v["key"] == _|_ {
        								key: envName
        							}
        						}
        					}
        				},
        			]
        		}]
        	}
        }

        parameter: {
        	// +usage=The mapping of environment variables to secret
        	envMappings: [string]: [string]: string
        }
```

**Comment by zzxwill** [comment]
@wangyikewxgm It seems that you fixed a similar one, So you can take up this feature if you like:)

**Comment by wonderflow** [comment]
the labels and annotations trait also can't be showed.

\cc @yangsoon

**Comment by xiqiu618** [comment]
reasons for not showing is trail discovery is vela-system, switching to vela-system verification is normal

![image](https://user-images.githubusercontent.com/73998049/115110908-3b20d500-9fb0-11eb-88d5-d2208f4aa828.png)

![Uploading image.png…]()

**Comment by xiqiu618** [comment]
I can improve it  /assign

**Comment by xiqiu618** [comment]
it may involve the priority. when the trait has the same name, the definition local env has priority over to  in vela-system

**Comment by zzxwill** [comment]
Per the offline discussion with @xiqiu618 and my verification, xiqiu pointed out the root cause. Thanks. Trying to fix it in #1521.
